libraries/SITL/SIM_JSON.h

```diff
@@ -105,6 +109,8 @@ class JSON : public Aircraft {
         { "", "rng_4", &state.rng[3], DATA_FLOAT, false },
         { "", "rng_5", &state.rng[4], DATA_FLOAT, false },
         { "", "rng_6", &state.rng[5], DATA_FLOAT, false },
+        {"windvane","direction", &state.wind_vane_apparent.direction, DATA_FLOAT, false},
```

Comment:
can't this be just airspeed ?

Reply:
In theory Yes, but there is no reason we cant have both, I think this avoids confusion.